.github/workflows/pr.yml

```diff
@@ -24,3 +23,40 @@ jobs:
         run: yarn test
       - name: yarn test:e2e
         run: yarn test:e2e
+  integration:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v4
+      - name: Use Node.js 18
+        uses: actions/setup-node@v4
+        with:
+          node-version: "18"
+      - name: yarn install
+        run: yarn
+      - name: yarn start:docker
+        run: yarn start:docker
+      - name: yarn build
+        run: yarn build
+      # Borrowed from https://github.com/DCFoundation/cosmos-proposal-builder/blob/main/.github/workflows/pr.yml#L43-L61
+      - name: Wait for Docker container to be ready
+        run: |
+          timeout 180 bash -c '
+            TARGET_HEIGHT=1001
```

Comment:
This was a bit of a hack to make sure all of the (replay?) is complete before submitting new tx's. This number might be a bit higher now with the addition of more proposals to a3p. I'm not sure of a good way to make this more dynamic, or if a different solution could be sourced from a3p.

Reply:
> ... or if a different solution could be sourced from a3p.

Yeah I think so, filed https://github.com/Agoric/agoric-3-proposals/issues/130